proto/connect/organisation_service/v1alpha1/organisation_service.proto

```diff
@@ -0,0 +1,34 @@
+// Copyright 2025 Cofide Limited.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package proto.connect.organisation_service.v1alpha1;
+
+import "proto/organisation/v1alpha1/organisation.proto";
+
+option go_package = "github.com/cofide/cofide-api-sdk/gen/go/proto/connect/organisation_service/v1alpha1";
+
+service OrganisationService {
+  rpc GetOrganisation(GetOrganisationRequest) returns (GetOrganisationResponse);
+  rpc ListOrganisations(ListOrganisationsRequest) returns (ListOrganisationsResponse);
+}
+
+message GetOrganisationRequest {
+  optional string organisation_id = 1;
```

Comment:
I have used org_id when referencing organisations as a parent of other resources. It might be more consistent to do the same here?